programs/clearing_house/src/controller/liquidation.rs

```diff
@@ -593,11 +600,30 @@ pub fn liquidate_borrow(
         liability_price,
     )?;
 
+    let liability_transfer_for_user: u128;
     {
         let mut liability_bank = bank_map.get_ref_mut(&liability_bank_index)?;
 
+        // part liquidator liability transfer pays to insurance fund
+        // size will be eventually be 0 for sufficiently small liability size
+        let liability_transfer_for_insurance = liability_transfer
```

Comment:
might be nice to delete this for now since it's not used

Reply:
what do you mean "its not used"?